src/pages/admin/reports/components/funds-expenditures-section/components/common/delete-funds-expenditures-category-modal/DeleteFundsExpendituresCategoryModal.module.scss

```
@@ -107,10 +122,14 @@
     width: 100%;
 }
 
+.panel :global(.hint-box) {
```

Stylelint (17.10.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)